website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md

```diff
@@ -64,65 +64,26 @@ echo $KARPENTER_VERSION $CLUSTER_NAME $AWS_DEFAULT_REGION $AWS_ACCOUNT_ID
 
-**Example 2: Create basic cluster with Karpenter on Fargate**
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh" language="bash"%}}
-
-Karpenter itself can run anywhere, including on [self-managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/worker.html), [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html) (Example 1), or [AWS Fargate](https://aws.amazon.com/fargate/)(Example 2).
-
-Karpenter will provision EC2 instances in your account.
-
-### Create the Karpenter Infrastructure and IAM Roles
-
-Karpenter requires IAM permissions to launch and connect instances and requires infrastructure to monitor [interruption events]({{<ref "../../concepts/deprovisioning/#interruption" >}}). This command provisions the relevant infrastructure and IAM roles using Cloudformation.
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step03-iam-cloud-formation.sh" language="bash"%}}
-
-### Grant Access to Nodes to Join the Cluster
-
-Instances launched by Karpenter must run with an InstanceProfile that grants permissions necessary to run containers and configure networking. Karpenter discovers the InstanceProfile using the name `KarpenterNodeRole-${ClusterName}`. This command adds the Karpenter node role to your aws-auth configmap, allowing nodes to connect.
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step04-grant-access.sh" language="bash"%}}
-
-Now, Karpenter can launch new EC2 instances and those instances can connect to your cluster.
-
-### Create the KarpenterController IAM Role
-
-Karpenter requires permissions like launching instances. This will create an AWS IAM Role, Kubernetes service account, and associate them using [IRSA](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html).
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step05-controller-iam.sh" language="bash"%}}
-
-### Create the EC2 Spot Service Linked Role
-
-This step is only necessary if this is the first time you're using EC2 Spot in this account. More details are available [here](https://docs.aws.amazon.com/batch/latest/userguide/spot_fleet_IAM_role.html).
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step06-add-spot-role.sh" language="bash"%}}
-
-### Install Karpenter Helm Chart
-
-Use Helm to deploy Karpenter to the cluster.
-
-Install the chart passing in the cluster details and the Karpenter role ARN.
-
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step08-apply-helm-chart.sh" language="bash"%}}
-
-### Optional Configuration
+## 4. Configure Optional Features
```

Comment:
Thoughts on moving this to the end, so users can get started w/ provisioners & deployments quickly?

* Create Provisioner
* Scale Up Deployment
* Scale Down Deployment
Advanced Usage
* Monitor with Grafana

Reply:
Done.